src/components/standalone/security/ips/IpsDisabledRules.vue

```diff
@@ -0,0 +1,221 @@
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import IpsDisableRuleDrawer from '@/components/standalone/security/ips/IpsDisableRuleDrawer.vue'
+import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
+import IpsEnableRuleModal from '@/components/standalone/security/ips/IpsEnableRuleModal.vue'
+
+export type Rule = {
+  description: string
+  id: string
+  gid: string
+  sid: string
+}
+
+type ListRuleResponse = AxiosResponse<{
+  rules: Rule[]
+}>
+
+const changes = useUciPendingChangesStore()
+
+const { t } = useI18n()
+
+const error = ref<Error>()
+const loading = ref(false)
+const rules = ref<Rule[]>([])
+
+function listDisabledRules() {
+  ubusCall('ns.snort', 'list-disabled-rules', {})
+    .then((response: ListRuleResponse) => {
+      rules.value = response.data.rules
+    })
+    .catch((reason) => {
+      error.value = reason
+    })
+    .finally(() => {
+      loading.value = false
+    })
+}
+
+onMounted(() => {
+  listDisabledRules()
+})
+
+const filter = ref('')
+const filteredRules = computed((): Rule[] => {
+  return rules.value.filter((rule) => {
+    return rule.description.includes(filter.value) || rule.id.includes(filter.value)
+  })
+})
+
+const sortKey = ref<keyof Rule>('id')
+const sortDescending = ref(false)
+const { sortedItems } = useSort(filteredRules, sortKey, sortDescending, {})
+
+const pageSize = ref(10)
+const { currentPage, paginatedItems } = useItemPagination(sortedItems, {
+  itemsPerPage: pageSize
+})
+
+// FIXME: when types from library are fixed, use proper type
+const onSort = (payload: any) => {
+  sortKey.value = payload.key
+  sortDescending.value = payload.descending
+}
+
+const disablingRule = ref(false)
+function handleSave() {
+  listDisabledRules()
+  changes.getChanges()
+  disablingRule.value = false
+}
+
+function dropDownActions(rule: Rule): NeDropdownItem[] {
+  return [
+    {
+      id: 'delete',
+      label: t('common.delete'),
+      icon: 'trash',
+      iconStyle: 'fas',
+      danger: true,
+      action: () => {
+        ruleToEnable.value = rule
+      }
+    }
+  ]
+}
+const ruleToEnable = ref<Rule>()
+function handleEnabled() {
+  listDisabledRules()
+  changes.getChanges()
+  ruleToEnable.value = undefined
+}
+</script>
+
+<template>
+  <div class="space-y-8">
+    <div class="flex flex-wrap items-start justify-between gap-4">
+      <p class="max-w-lg">{{ t('standalone.ips.disabled_rules_description') }}</p>
+      <IpsEnabledBadge />
+    </div>
+
+    <NeInlineNotification
+      v-if="error"
+      :description="t(getAxiosErrorMessage(error))"
+      :title="t('standalone.ips.error_loading_disabled_rules')"
+      kind="error"
+    />
+    <NeSkeleton v-if="loading" :lines="10" />
+    <template v-else-if="rules.length > 0">
+      <div class="flex flex-col flex-wrap justify-between gap-4 md:flex-row">
+        <NeTextInput v-model="filter" :placeholder="t('common.filter')">
```

Comment:
```suggestion
        <NeTextInput v-model.trim="filter" :placeholder="t('common.filter')" is-search class="max-w-xs sm:max-w-sm">
```
`prefix` slot is not needed if we use `is-search`. This way we also get the little cross button to clear the filter